cfme/tests/infrastructure/test_host.py

```diff
@@ -248,3 +248,68 @@ def test_250_vmware_hosts_loading(appliance, create_250_hosts, view_type):
     view.entities.paginator.set_items_per_page(1000)
     view.toolbar.view_selector.select(view_type)
     wait_for(view.entities.get_first_entity, timeout=60, message='Wait for the view')
+
+
+@test_requirements.general_ui
```

Comment:
**Question** Any BZ associated with this?

Reply:
I'm not really sure. I picked this test from `test_manual.py` ,there was no BZ mentioned.